packages/presets/src/tests/test_erc721.cairo

```diff
@@ -513,38 +504,6 @@ fn test_safeTransferFrom_to_account() {
     assert_state_after_transfer(dispatcher, owner, account, token_id);
 }
 
-#[test]
-fn test_safe_transfer_from_to_account_camel() {
-    let (mut spy, dispatcher) = setup_dispatcher();
-    let account = setup_camel_account();
-    let token_id = TOKEN_1;
-    let owner = OWNER();
-    spy.drop_all_events();
-
-    assert_state_before_transfer(dispatcher, owner, account, token_id);
-
-    dispatcher.safe_transfer_from(owner, account, token_id, DATA(true));
-    spy.assert_only_event_transfer(dispatcher.contract_address, owner, account, token_id);
-
-    assert_state_after_transfer(dispatcher, owner, account, token_id);
-}
-
-#[test]
-fn test_safeTransferFrom_to_account_camel() {
-    let (mut spy, dispatcher) = setup_dispatcher();
```

Comment:
There's no difference to the erc721 module, and there was not really a difference before either, since the camel account target is just another contract exposing `supports_interface`, since there's no camelCase version of SRC5.